src/engine/engine_bf.c

```diff
@@ -51,25 +51,25 @@ static void _ocf_backfill_complete(struct ocf_request *req, int error)
 	 * request. Also, complete original request only if this is the last
 	 * sub-request to complete
 	 */
-	if (env_atomic_dec_return(&req->req_remaining) == 0) {
-		/* We must free the pages we have allocated */
-		ctx_data_secure_erase(cache->owner, req->data);
-		ctx_data_munlock(cache->owner, req->data);
-		ctx_data_free(cache->owner, req->data);
-		req->data = NULL;
-
-		if (req->error) {
-			env_atomic_inc(&cache->core[req->core_id].
-					counters->cache_errors.write);
-			ocf_engine_invalidate(req);
-		} else {
-			ocf_req_unlock(req);
-
-			/* always free the request at the last point
-			 * of the completion path
-			 */
-			ocf_req_put(req);
-		}
+	if (env_atomic_dec_return(&req->req_remaining))
+		return;
+
+	/* We must free the pages we have allocated */
+	ctx_data_secure_erase(cache->owner, req->data);
+	ctx_data_munlock(cache->owner, req->data);
+	ctx_data_free(cache->owner, req->data);
+	req->data = NULL;
+
+	if (req->error) {
+		env_atomic_inc(&req->core->counters->cache_errors.write);
+		ocf_engine_invalidate(req);
+	} else {
+		ocf_req_unlock(req);
+
+		/* always free the request at the last point
```

Comment:
Can you move more text to this line to fill 80 char width?

Reply:
Done. After some rephrasing it actually fits into 80 chars.